src/expr.cpp

```diff
@@ -1826,32 +1826,21 @@ bool lCreateBinaryOperatorCall(const BinaryExpr::Op bop, Expr *a0, Expr *a1, Exp
     }
     if (CastType<StructType>(type0) != nullptr || CastType<StructType>(type1) != nullptr) {
         std::string opName = std::string("operator") + lOpString(bop);
-        std::vector<Symbol *> funs;
-        m->symbolTable->LookupFunction(opName.c_str(), &funs);
-        if (funs.size() > 0) {
-            Expr *func = new FunctionSymbolExpr(opName.c_str(), funs, sp);
-            ExprList *args = new ExprList(sp);
-            args->exprs.push_back(arg0);
-            args->exprs.push_back(arg1);
-            op = new FunctionCallExpr(func, args, sp);
-            return abort;
-        }
```

Comment:
There was a duplicate code (same is on lines 1832-1841) - one for functions and another for function templates. Now since we're passing both functions and function templates to FunctionSymbolExpr constructor we can keep only one.